decidim-assemblies/app/controllers/decidim/assemblies/assemblies_controller.rb

```diff
@@ -5,7 +5,9 @@ module Assemblies
     # A controller that holds the logic to show Assemblies in a public layout.
     class AssembliesController < Decidim::Assemblies::ApplicationController
       include ParticipatorySpaceContext
+
```

Comment:
I would avoid changing files unless it is necessary. Here you probably did some changes at first and then reverted the changes which caused this issue but if you revert stuff, revert back to the original in the future please. :pray: